File: langgraph/result/qwen/32b-py2js/125.js
function splitWords(txt) {
    if (txt.includes(" ")) {
        return txt.split(" ");
    } else if (txt.includes(",")) {
        return txt.replace(/,/g, ' ').split(" ");
    } else {
        return [...txt].filter(i => i >= 'a' && i <= 'z' && i.charCodeAt(0) % 2 === 0).length;
    }
}

File: langgraph/data/semantic_equiv/125.py
def partition_text(input_string):
    if ' ' in input_string:
        return input_string.split()
    elif ',' in input_string:
        modified_string = input_string.replace(',', ' ')
        return modified_string.split()
    else:
        lowercase_even_count = sum(1 for char in input_string if char.islower() and ord(char) % 2 == 0)
        return lowercase_even_count

# method name: split_words -> partition_text
# parameter name: txt -> input_string
# replaced list comprehension with sum for counting lowercase even characters
# renamed variable for clarity: counts -> lowercase_even_count